XcodeServerSDK/Server Entities/Issue.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Issue.swift
+//  XcodeServerSDK
+//
+//  Created by Mateusz Zając on 04.08.2015.
+//  Copyright © 2015 Honza Dvorsky. All rights reserved.
+//
+
+import Foundation
+
+public class Issue: XcodeServerEntity {
+    
+    public enum Type: String {
+        case BuildServiceError = "buildServiceError"
+        // Looks like build service warnings are treated as warnings in case
+        // of type. In case of Issue Type they're Build Service Error
+        // case BuildServiceWarining = "warning"
+        case TriggerError = "triggerError"
+        case Error = "error"
+        case Warning = "warning"
+        case TestFailure = "testFailure"
+        case AnalyzerWarning = "analyzerWarning"
```

Comment:
I don't think that `resolved` is a type of `Issue`. From response JSONs I understand that it's a category of `Issues` equal to those listed in `IssueStatus`. And yet, we still can't do anything about `buildServiceWarnings`... No clear explanation how to determine them from regular `warnings`.